mod_conf_sql.c

```diff
@@ -157,6 +157,12 @@ static int sqlconf_parse_uri_db(char **uri) {
   *tmp = '\0';
   sqlconf_db.database = pstrdup(sqlconf_pool, *uri);
 
+  int i;
+  int n=strlen(sqlconf_db.server);
+
+  // replace \ with /
+  for(i=0;i<=n;i++) if(sqlconf_db.server[i]=='\\') sqlconf_db.server[i]='/';
```

Comment:
idealy the url while be : 
sql://dbidrivername/path/to/dabasefile.db?dbpass=pass&dbuser=user&dbhost=hostname&dbctxt=ftpctxt&conf=ftpconf&map=ftpmap

exemple for sqlite:
sql://SQLite/path/to/sqlitefile.db?dbctxt=ftpctxt&conf=ftpconf&map=ftpmap
exemple for mysql:
sql://mysql/databasename?dbuser=user&dbpass=pass&dbhost=hostname&dbctxt=ftpctxt&conf=ftpconf&map=ftpmap